api-server/src/events/dto/create-event-request.ts

```diff
@@ -87,7 +87,7 @@ export class CreateEventRequest {
 
   @ApiProperty({ example: 'radio-mc-radio-station' })
   @IsOptional()
-  reviewed_by_org: boolean;
+  reviewed_by_org: string;
```

Comment:
this was something else I noticed while working on this, this type was simply wrong